docs/source/publish_and_share_kedro_viz_on_aws.md

```diff
@@ -0,0 +1,118 @@
+
+```bash
+pip install -r src/requirements.txt
+```
+
+Kedro-Viz requires specific minimum versions of `fsspec[s3]`, and `kedro` to publish your project. Ensure you have these correct versions by updating the `requirements.txt` file of the Kedro project to add the following:
+
+```text
+fsspec[s3]>=2023.9.0
+kedro>=0.18.2
+```
+
+## Install dependencies
+Step 1:
+
+```bash
+pip install 'kedro-viz[aws]'
+```
+
+## Configure your AWS S3 bucket
+
+Step 2: You can host your Kedro-Viz project on Amazon S3. You must first create an S3 bucket and then enable static website hosting. 
+
+To do so, follow the [AWS tutorial](https://docs.aws.amazon.com/AmazonS3/latest/userguide/HostingWebsiteOnS3Setup.html) to configure a static website on Amazon S3.
+
+```{note}
+Uploading your site's files will be done through Kedro-Viz
+```
+
+Step 3: Once the S3 bucket is created, you'll need to create an Identity and Access Management (IAM) user account, user group, and generate the corresponding access keys. To do so:
+
+Step 4: Sign in to the [AWS Management Console](https://console.aws.amazon.com/s3/) and create an IAM user account.
+For more information, see the official AWS documentation about [IAM Identities](https://docs.aws.amazon.com/IAM/latest/UserGuide/id.html).
+
+![](./images/kedro_viz_share_credentials1.png)
+
+Step 5: Create a user group from the IAM dashboard, ensuring the user group has full access to the AWS S3 policy.
+
+![](./images/kedro_viz_share_credentials2.png)
+
+For more information, see the official AWS documentation about [IAM user groups](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_groups.html?icmpid=docs_iam_help_panel).
+
+Step 6: Add the IAM user to the user group (this is only possible if the group has been created).
+
+![](./images/kedro_viz_share_credentials3.png)
+
+Step 7: Select the user, then select `Create access key`. Follow the steps and create your keys.
+
+![](./images/kedro_viz_share_credentials4.png)
+
+### Permissions and access control
+
+All permissions and access control are controlled by AWS. It's up to you, the user, if you want to allow anyone to see your project or limit access to certain IP addresses, users, or groups.
+
+You can control who can view your visualisation using [bucket and user policies](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-iam-policies.html) or [access control lists](https://docs.aws.amazon.com/AmazonS3/latest/userguide/acls.html). See the official AWS documentation for more information.
+
+## Set credentials
+
+Step 8: Once that's completed, you'll need to set your AWS credentials as environment variables in your terminal window, as shown below:
+
+```bash
+export AWS_ACCESS_KEY_ID="your_access_key_id"
+export AWS_SECRET_ACCESS_KEY="your_secret_access_key"
+```
+
+For more information, see the official AWS documentation about [how to work with credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html).
+
+
+### Billing
```

Comment:
Why have this section on Billing? That has nothing to do with Kedro-Viz or Kedro, so I don't really see why it should be added.